modules/platforms/cpp/odbc/CMakeLists.txt

```diff
@@ -102,6 +102,52 @@ else()
     target_link_libraries(${TARGET} ignite-common ignite-binary ignite-network)
 endif()
 
+if (WIN32)
+	set(WIX_ODBC_LIBRARY_PATH ".\\ignite.odbc.dll")
+
+	set(WIX_PROJECT_NAME "Apache Ignite")
+	set(WIX_MANUFACTURER "The Apache Software Foundation")
+	set(WIX_COPYRIGHT_COMMENT "Apache, Apache Ignite, the Apache feather and the Apache Ignite logo are trademarks of The Apache Software Foundation.")
+
+	if(CMAKE_SIZEOF_VOID_P EQUAL 8)
+		set(WIX_BIT_SUFFIX "64-bit")
+		set(WIX_PRODUCT_ID "F3E308E4-910C-4AF5-82DE-2ACF4D64830E")
+		set(WIX_UPGRADE_CODE "1D7AEFDF-6CD2-4FB5-88F2-811A89832D6D")
+		set(WIX_COMPONENT_ID "E5F0DDF2-DD3C-4196-8A08-70921858A52F")
+		set(WIX_PROGRAM_FILES_FOLDER_ID "ProgramFiles64Folder")
+		set(WIX_COMPONENT_IS_WIN64 "yes")
+		set(WIX_PACKAGE_PLATFORM "x64")
+		set(WIX_INSTALLER_PREFIX ${CMAKE_BINARY_DIR}/${CMAKE_BUILD_TYPE}/ignite-odbc-amd64)
+	elseif(CMAKE_SIZEOF_VOID_P EQUAL 4)
+		set(WIX_BIT_SUFFIX "32-bit")
+		set(WIX_PRODUCT_ID "D39CBABA-1E21-4701-AA5C-91EDA07B383B")
+		set(WIX_UPGRADE_CODE "743902A4-365C-424E-B226-5B2898A3941E")
+		set(WIX_COMPONENT_ID "4AFA26EE-C639-4EF2-A9B2-281119BB4BB5")
+		set(WIX_PROGRAM_FILES_FOLDER_ID "ProgramFilesFolder")
+		set(WIX_COMPONENT_IS_WIN64 "no")
+		set(WIX_PACKAGE_PLATFORM "x86")
+		set(WIX_INSTALLER_PREFIX ${CMAKE_BINARY_DIR}/${CMAKE_BUILD_TYPE}/ignite-odbc-x86)
+	endif()
+
+	set(WIX_PRODUCT_NAME "${WIX_PROJECT_NAME} ODBC ${WIX_BIT_SUFFIX} Driver")
+
+	configure_file(
+        "${CMAKE_CURRENT_LIST_DIR}/install/ignite-odbc.wxs.in"
+        "${WIX_INSTALLER_PREFIX}.wxs"
+		@ONLY
+	)
+
+	add_custom_command(
+		TARGET ${TARGET} POST_BUILD
+		COMMAND candle ${WIX_INSTALLER_PREFIX}.wxs -out ${WIX_INSTALLER_PREFIX}.wxobj
+		COMMAND light -ext WixUIExtension ${WIX_INSTALLER_PREFIX}.wxobj -out ${WIX_INSTALLER_PREFIX}.msi
+	)
```

Comment:
Will building of ODBC fail if user does not have WiX installed?

Reply:
Igor, yep, I'will add optional flag for it, soon.